cartography/intel/azure/iam.py

```diff
@@ -260,24 +261,54 @@ def get_tenant_groups_list(client: GraphRbacManagementClient, tenant_id: str) ->
 
 
 def _load_tenant_groups_tx(
-    tx: neo4j.Transaction, tenant_id: str, tenant_groups_list: List[Dict], update_tag: int,
+    tx: neo4j.Transaction,
+    tenant_id: str,
+    tenant_groups_list: List[Dict],
+    update_tag: int,
 ) -> None:
+    """Load Azure groups into Neo4j"""
     ingest_group = """
     UNWIND $tenant_groups_list AS group
     MERGE (i:AzureGroup{id: group.id})
     ON CREATE SET i:AzurePrincipal,
-    i.firstseen = timestamp(),
-    i.object_id=group.object_id,
-    i.region = $region,
-    i.create_date = $createDate,
-    i.name = group.display_name,
-    i.visibility = group.visibility,
-    i.classification = group.classification,
-    i.createdDateTime = group.createdDateTime,
-    i.consolelink = group.consolelink,
-    i.securityEnabled = group.security_enabled
+    i.firstseen = timestamp()
     SET i.lastupdated = $update_tag,
-    i.mail = group.mail
+        i.object_id = group.id,
+        i.deleted_date_time = group.deletedDateTime,
+        i.classification = group.classification,
+        i.created_date_time = group.createdDateTime,
```

Comment:
create_date is missing. is it used anywhere? if yes, can we calculate date from the createdDateTime field?